src/schema/meta/context.yaml

```diff
@@ -0,0 +1,137 @@
+#
+# The context provides the namespaces available to rules.
+# These namespaces are used by selectors to define the scope of application
+# for a rule, as well as assertions, to determine whether the rule is satisfied.
+#
+# Not all components of the context will be defined; for example a NIfTI header
+# will only be defined when examining a NIfTI file.
+#
+# The dataset namespace is constructed once and is available when visiting all
+# files.
+#
+# The subject namespace is constructed once per subject, and is available when
+# visiting all files within that subject.
+#
+# All other (current) namespaces are defined on individual files.
+# Sidecar metadata and file associations are built according to the inheritance
+# principle.
+#
+---
+context:
+    dataset:
```

Comment:
I believe we also use `dataset.modalities`, but I guess we might want to access that through `subject` or `session`...